net-libs/pacparser/pacparser-1.3.7-r1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit eutils python-r1 toolchain-funcs
+
+DESCRIPTION="Library to parse proxy auto-config files"
+HOMEPAGE="http://pacparser.manugarg.com/"
```

Comment:
https?

Reply:
no clue, please diff that -r1 with the original and do not ask me about code from others ... like the eapi